Misc/NEWS.d/next/Library/2022-08-23-03-13-18.gh-issue-96192.TJywOF.rst

```diff
@@ -0,0 +1 @@
+Fixed the :func:`os.ismount()` to use a path that is str or bytes in all cases.
```

Comment:
```suggestion
Fix handling of ``bytes`` :term:`path-like objects <path-like object>` in :func:`os.ismount()`.
```

Reply:
Updated in the current PR, added you as Co-Author.